packages/manager/apps/pci-private-registry/src/constants.ts

```diff
@@ -10,3 +23,26 @@ export const PRIVATE_REGISTRY_STATUS = {
   DELETING: 'DELETING',
   SCALING_UP: 'SCALING_UP',
 };
+
+export const GUIDES = [
+  {
+    id: 'configure',
+    link: 'https://docs.ovh.com/',
+  },
+  {
+    id: 'registry',
+    link: 'https://docs.ovh.com/',
+  },
+  {
+    id: 'deploy',
+    link: 'https://docs.ovh.com/',
+  },
+];
+export const DELETE_CONFIRMATION_INPUT = /^DELETE$/;
+
+export default {
+  PRIVATE_REGISTRY_STATUS_MAP,
+  PRIVATE_REGISTRY_STATUS,
+  GUIDES,
+  DELETE_CONFIRMATION_INPUT,
+};
```

Comment:
Dont understand why you export en _default_, since your variables are already exported on _named export_